### ISSUE TITLE
Add example for running embedded inside a larger app

### ISSUE THREAD

**Issue by johnwiseheart** [theia-ide/theia#3823]
I am trying to include Theia embedded inside a larger app (without using iframes). To do this, I am extending `FrontendApplication` and overriding the host. I ran into some issues with `style-loader/useable`, so I decided to extend the `BuiltinThemeProvider` and several other theme related sections such that I can include the CSS myself. I'm now running into issues with bindings (likely because of how I have overridden everything else).

It would be great if there was an example on how to embed this in a larger application. My use case is that I have a custom way to render a variety of react "plug-ins" as components on a single page - and so I'm looking for a way to be able to pass Theia an `Element` and have it render to that.

**Comment by johnwiseheart** [comment]
Sure - happy to do that! This I guess explains why I’ve found it hard to find any existing documentation. I have run into a variety of errors along the way - probably because of my webpack config. Would my best method for solving these to be create existing issues for each and then working with you folks through them? Or is there a more better approach I could take?

**Comment by svenefftinge** [comment]
Yes, that would be a valid way. You can also chime in on the chat and ask.

**Comment by svenefftinge** [comment]
Also just noticed https://github.com/theia-ide/theia/issues/1045#issuecomment-446909231

**Comment by talsenteam** [comment]
Hi there,
Is there already an example for integration of theia into (e.g.) an angular project?
I tried this myself, but it unfortunately did not work. Building the application was kind of strange, when both the theia core module and angular modules are in the same package.json file.
If there in not yet a solution, I would be interested on working together (pair- / mob-programming) for a solution.
You can reach me at github@talsen.team and suggest date and time for a meeting (I'm from Austria => UTC+02), I also joined your spectrum.chat as user talsen-team-gmbh.
I'm looking forward to the pair- / mob-programming :)

**Comment by akosyakov** [comment]
@talsenteam I don't think so. If you come up with something let us know we can have such project under https://github.com/theia-ide and add you as a collaborator there.

**Comment by talsenteam** [comment]
@akosyakov Ok. I'll try to prepare a (very simplistic) angular project and also some docker build + docker compose configuration for it, where we can start from. Ok?

**Comment by quantumsheep** [comment]
Is there any update on this?
